plugins/zfs/zpool_capacity

```diff
@@ -0,0 +1,259 @@
+            critical ${critical}
+        "
+        for p in $pool_list
+        do
+            data_attr="${data_attr}
+                ${p} GAUGE LINE2 ${p}"
+        done
+        ;;
+    allocated)
+        global_attr="
+            graph_title ZFS storage pool - Allocated bytes
+            graph_category fs
+            graph_args --base 1024 --lower-limit 0
+            graph_vlabel Bytes
+            graph_info ZFS storage pool - Allocated bytes
+        "
+        c=0
+        for p in $pool_list
+        do
+            data_attr="${data_attr}
+                ${p}_size      GAUGE LINE  ${p} size
+                ${p}_allocated GAUGE LINE2 ${p} allocated"
+            global_attr="${global_attr}
+                ${p}_size.colour      COLOUR${c}
+                ${p}_allocated.colour COLOUR${c}"
+            c=$(( c + 1 ))
+        done
+        ;;
+    dedup)
+        global_attr="
+            graph_title ZFS storage pool - Dedup and compress ratio
+            graph_category fs
+            graph_args --base 1000 --lower-limit 1
+            graph_vlabel Ratio
+            graph_info ZFS storage pool - Dedup and compress ratio
+        "
+        for p in $pool_list
+        do
+            data_attr="${data_attr}
+                ${p}_dedup    GAUGE LINE ${p} dedup
+                ${p}_compress GAUGE LINE ${p} compress"
+        done
+        ;;
+    esac
+}
+
+do_config() {
+    local func=$1
+    local label_max_length=45 
+    local field type draw label
+
+    preconfig "$func"
+    echo "multigraph ${plugin_name}_${func}"
+
+    # print global attributes
+    echo "$global_attr" | sed -e 's/^  *//' -e '/^$/d'
+
+    # print data source attributes
+    echo "$data_attr" | while read -r field type draw label
+    do
+        [ -z "$field" ] && continue
+
+        field=$( clean_fieldname "$field" )
+        echo "${field}.type ${type}"
+        echo "${field}.draw ${draw}"
+        echo "${field}.label ${label:0:${label_max_length}}"
+        if [ "$type" = 'DERIVE' ]; then
+            echo "${field}.min 0"
+        fi
+        if [ "$label" = 'dummy' ]; then
+            echo "${field}.graph no"
+        fi
+    done
+
+    echo
+}
+
+get_stats() {
+    local func=$1
+
+    case $func in
+    capacity)
+        $zpool_cmd list -H -o name,capacity | sed 's/%$//'
+        ;;
+    allocated)
+        ( $zpool_cmd list -H -o name,allocated \
+          | awk '{ print $1"_allocated", $2 }'
+          $zpool_cmd list -H -o name,size \
+          | awk '{ print $1"_size", $2 }'
+        ) \
+        | perl -ane '
+            @unit{ qw/ K M G T P E / } = ( 1 .. 6 );
+            $name = $F[0];
+            $byteu = $F[1];
+            ( $n, $u ) = $byteu =~ /^([\d.]+)([KMGTPE]?)$/;
+            $byte  = int( $n * 1024 ** ( $u ? $unit{ $u } : 0 ) );
+            print "$name $byte\n";
+        '
+        # Note: ZFS supports up to 16EB.
+        ;;
+    dedup)
+        $zpool_cmd list -H -o name,dedup \
```

Comment:
Quotes, please.

Addittionally: please add an example output line of `-o name, dedup`.